index.html

```diff
@@ -75,7 +83,7 @@ <h1 class="works-title">My Recent Works</h1>
             </div>
             
             <div class="first-card">
```

Comment:
- [ ] Your works section is almost ready, let's try to make it even better, like the Figma template:

![Captura de pantalla 2022-03-25 a la(s) 7 46 37 p  m](https://user-images.githubusercontent.com/83717285/160218643-5f3bb732-392c-413d-b4a6-557369c63b5b.png)

- Please add some space around the title and line.
- Kindly reduce the space between the image and the text info.